cli/cli/helpers/engine_manager/engine_existence_guarantor.go

```diff
@@ -261,3 +279,29 @@ func (guarantor *engineExistenceGuarantor) getRunningAndCLIEngineVersions() (*se
 
 	return runningEngineSemver, launcherEngineSemver, nil
 }
+
+func (guarantor *engineExistenceGuarantor) ensureCentralizedLogsComponentsAreRunning(ctx context.Context, shouldForceContainerRestart bool) error {
+
+	logsDatabase, err := guarantor.kurtosisBackend.GetLogsDatabase(ctx)
```

Comment:
Still very much confused by the wording "database" VS "aggregator" here. Is it just me or should we harmonize things a bit more?